wandb/sdk/data_types/object_3d.py

```diff
@@ -232,15 +241,31 @@ def from_file(
 
     @classmethod
     def from_numpy(cls, data: "np.ndarray") -> "Object3D":
+        """Initializes Object3D from a numpy array.
+
+        Arguments:
+            data (numpy array): Each entry in the array will
+                represent one point in the point cloud.
+
+
+        The shape of the numpy array must be one of either:
+        ```
+        [[x y z],       ...] nx3
+        [[x y z c],     ...] nx4 where c is a category with supported range [1, 14]
+        [[x y z r g b], ...] nx6 where is rgb is color
```

Comment:
```suggestion
        [[x y z],       ...]  # nx3.
        [[x y z c],     ...]  # nx4 where c is a category with supported range [1, 14].
        [[x y z r g b], ...]  # nx6 where is rgb is color.
```

Reply:
For my own learning: what's the purpose of the # ? As someone mostly unfamiliar with python docs I don't know the convention around that.